### PR TITLE
Added internal auth for cip-email-verification

### PR DESCRIPTION
**Pull request by mar36tin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,18 +49,20 @@ For reference here are the details for running each of the services individually
 
 #### Verify
 
-    -XPOST -H "Content-type: application/json" -d '{
+TY8wpfCH-cyTL8mBiAsoPlbMW5eNYcdONHb0wNCWxlkP18UJoiIqBtbWbXCzgnlKWylcODd5y
```
Comment (johno1985) [inline README.md:52]: typo?

**Review by johno1985 — COMMENTED** [review]
comment added

File: app/uk/gov/hmrc/cipemailverification/controllers/InternalAuthAccess.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+package uk.gov.hmrc.cipemailverification.controllers
+
+import uk.gov.hmrc.internalauth.client.{IAAction, Predicate, Resource, ResourceLocation, ResourceType}
+
+object InternalAuthAccess {
+  val permission: Predicate.Permission = Predicate.Permission(Resource(
+    ResourceType("cip-email-verification"),
+    ResourceLocation("")),
```
Comment (johno1985) [inline app/uk/gov/hmrc/cipemailverification/controllers/InternalAuthAccess.scala:24]: I think this should be "*"

File: test/requests.rest [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ### Correct-email
 POST http://localhost:6183/customer-insight-platform/email/verify
 content-type: application/json
+Authorization: TY8wpfCH-cyTL8mBiAsoPlbMW5eNYcdONHb0wNCWxlkP18UJoiIqBtbWbXCzgnlKWylcODd5y
```
Comment (johno1985) [inline test/requests.rest:4]: these should just be fake tokens as these are only used locally

File: app/uk/gov/hmrc/cipemailverification/controllers/InternalAuthAccess.scala (hunk 2 above)
Comment (mar36tin) [reply app/uk/gov/hmrc/cipemailverification/controllers/InternalAuthAccess.scala:24]: Actually this is one thing I wanted to speak to you about, basically i spoke to Gerald from platops and he said we should really leave it blank and put the action as READ

**Review by johno1985 — COMMENTED** [review]
minor comment

File: app/uk/gov/hmrc/cipemailverification/controllers/InternalAuthAccess.scala (hunk 2 above)
Comment (johno1985) [reply app/uk/gov/hmrc/cipemailverification/controllers/InternalAuthAccess.scala:24]: In here in the code, it should be "*" but in the internal-auth-config it is READ

Comment (johno1985) [reply app/uk/gov/hmrc/cipemailverification/controllers/InternalAuthAccess.scala:24]: Tested, works for validation service locally and in QA

**Review by johno1985 — APPROVED** [review]

**Pull request merged**